rules/properImportRequireRule.ts

```diff
@@ -0,0 +1,136 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as Lint from 'tslint';
+import * as ts from 'typescript';
+
+// True if es6 module
+const module_cache: { [k: string]: boolean } = {};
+
+function findModulePath(module_name: string): string {
+  let current_dir = process.cwd();
+  if (module_name.startsWith('./')) return ''; // Currently doesn't support local modules.
+
+  while (true) {
+    const current_path = path.join(current_dir, 'node_modules', module_name);
+    if (fs.existsSync(current_path)) return current_path;
+
+    const next_dir = path.dirname(current_dir);
+    if (current_dir === next_dir) return '';
+    current_dir = next_dir;
+  }
```

Comment:
should use `require.resolve` instead

Reply:
The node_modules folder isn't included in the require path.